client/cli/src/params/network_params.rs

```diff
@@ -85,9 +85,11 @@ pub struct NetworkParams {
 	#[structopt(flatten)]
 	pub node_key_params: NodeKeyParams,
 
-	/// Experimental feature flag.
-	#[structopt(long = "use-yamux-flow-control")]
-	pub use_yamux_flow_control: bool,
+	/// Disable the yamux flow control. Enable this if the networking seems broken. This option
+	/// will be removed in the future once there is enough confidence that this feature is
+	/// properly working.
+	#[structopt(long = "no-yamux-flow-control")]
```

Comment:
You can just write "long" without giving a value and structopt will automatically make the parameter based on the field name (changing _ to - automatically)

Reply:
```suggestion
	#[structopt(long)]
```